demos/common/python/tokens_bert.py

```diff
@@ -17,6 +17,37 @@
 import unicodedata
```

Comment:
I think this file fit better in the `models` dir.

Reply:
Moved to `models/`